config/application.php

```diff
@@ -155,7 +155,8 @@
 	'BlacksmithRankingLimit'	=> 20,						//
 	'HomunRankingLimit'			=> 20,						//
 	'MVPRankingLimit'			=> 20,						//
-
+	'MVPRankingMaxGroupID'		=> 99,						// Will hide <= Value (Value: 0 - 99 GroupID).
+	
 	'RankingHideGroupLevel'		=> AccountLevel::LOWGM,		//
```

Comment:
Perhaps the code could be made to honour the permissions that already exist? From what i recall, all the other ranking pages remove accounts with LOWGM and higher.